include/psen_scan/reply_msg.h

```diff
@@ -17,6 +17,7 @@
 #define PSEN_SCAN_REPLY_MSG_H
```

Comment:
Could we call it `StartReply` or `StartReplyMsg`? I think StartRequest and StartReply are names we can safely use.

I would also rename the file and adjust the header guards.